### ISSUE TITLE
Is there an alternative to paraview?

### ISSUE THREAD

**Issue by Haphisz** [dicengine/dice#228]
Hi, I am unable to load the DICe_solutiom.e file in paraview. Is there an alternative software to view the strain and displacement of the specimen? Thank you.

**Comment by Haphisz** [comment]
![Paraview display](https://user-images.githubusercontent.com/92328193/142695586-f6e1dabd-3467-4167-b860-6f6feafc72ee.PNG)

Could it be that there is a version of Paraview that works with DICe files, this is a screenshot of the interface after following the instructions from the video

**Comment by zhangkmsjtu** [comment]
> ![Paraview display](https://user-images.githubusercontent.com/92328193/142695586-f6e1dabd-3467-4167-b860-6f6feafc72ee.PNG)
> 
> Could it be that there is a version of Paraview that works with DICe files, this is a screenshot of the interface after following the instructions from the video

same with me!

**Comment by dicengine** [comment]
I would suggest viewing the results as a table in ParaView to make sure that the DICe solution is reasonable (for example, to make sure that all points didn't fail the correlation) You can also check this by looking at the sigma values in the .txt files. If the sigma value is -1, the point failed for some reason.

**Comment by Haphisz** [comment]
How do i adjust failed points, some of the sigma values have -1.

**Comment by dicengine** [comment]
If you only have a few points with sigma = -1 the rest of the points should show up fine in ParaView. Are you able to post your  DICe_solution.e file here so I can take a look at what's going on?

**Comment by Haphisz** [comment]
The .e file is not supported to be uploaded here.

**Comment by dicengine** [comment]
zip it

**Comment by Haphisz** [comment]
[DICe_solution.zip](https://github.com/dicengine/dice/files/7645476/DICe_solution.zip)

**Comment by dicengine** [comment]
I'm not sure what's going on with ParaView for you. The results file you sent opens just fine for me (see image below). I am using ParaView v5.5.2.
![Screen Shot 2021-12-03 at 6 37 05 AM](https://user-images.githubusercontent.com/15202746/144611455-8157b800-d5f7-4148-9092-4e63b06d882c.png)

**Comment by Haphisz** [comment]
Thank you for your help. I have downloaded paraview 5.5.2 and I can view the displacement contours now. They displacement units are in pixel right?

**Comment by dicengine** [comment]
Yes, DISPLACEMENT is in pixels MODEL_DISPLACEMENT is in physical units if you are using stereo.

**Comment by Haphisz** [comment]
Thanks, I am try to further manipulate my displacement data. Is it possible to get a displacement field data from paraview that can be exported to matlab?

**Comment by dicengine** [comment]
One option would be to use the DICe_solution_[step].txt files which are CSV and should be able to be easily read into matlab.

**Comment by Haphisz** [comment]
Yeah, I tried that. It reads has a column vector. I want to get the displacement as a array where each displacement value are present their exact location. I mean the values that produce the visualization on paraview.

**Comment by dicengine** [comment]
I know I have done that in the past with Matlab, but I think I had to interpret the .txt input as scatter data rather than a grid. The contour(X,Y,Z) function in matlab should work with scattered x,y points.

**Comment by Haphisz** [comment]
Thank you. After calling the contour(X,Y,Z) in matlab where X= Coordinate_X value, Y= Coordinate_Y value and Z in this case is the Displacement_Y value. I keep getting this error prompt:  **_Input arguments must be numeric or objects which can be converted to double._**. Which i think it's because X,Y are column vector and not a double matrix. Is there a way to fix this? Thank you once again.

**Comment by dicengine** [comment]
You might have to take out the headers in the first line that define the fields for each column.

**Comment by Haphisz** [comment]
Okay, but in this case, the Z data must be at least a 2x2 matrix.

**Comment by dicengine** [comment]
Read up on the documentation for Matlab, there are lots of functions in Matlab where z can be a vector like x and y, and doesn't have to be a matrix corresponding to regular grid point.

**Comment by gabrielegranello** [comment]
The Paraview team provided a solution here https://gitlab.kitware.com/paraview/paraview/-/issues/21469
